adding 'virtual network peering' resource (#421)
inspec/inspec-azure: docs/resources/azure_virtual_network_peerings.md

---
title: About the azure_virtual_network_peerings Resource
platform: azure
---

# azure_virtual_network_peerings

Use the `azure_virtual_network_peerings` InSpec audit resource to test properties related to virtual network peerings of a virtual network.

## Azure REST API version, endpoint and http client parameters

> This resource interacts with api versions supported by the resource provider.

Reviewer: Interacts is a vague word. Consider replacing with `uses` or another precise term.

The `api_version` can be defined as a resource parameter.
If not provided, the latest version will be used.
For more information, refer to [`azure_generic_resource`](azure_generic_resource.md).

Unless defined, `azure_cloud` global endpoint, and default values for the http client will be used.
For more information, refer to the resource pack [README](../../README.md). 

## Availability

### Installation

This resource is available in the [InSpec Azure resource pack](https://github.com/inspec/inspec-azure). 
For an example `inspec.yml` file and how to set up your Azure credentials, refer to resource pack [README](../../README.md#Service-Principal).

## Syntax

> The `resource_group` and `vnet` must be given as a parameter.

Reviewer: ```suggestion
This resource requires the `resource_group` and `vnet` parameters.
```

or
```
`resource_group` and `vnet` are required parameters.
```

Author: fixed
